setup.py

```diff
@@ -16,7 +16,7 @@
     long_description_content_type="text/markdown",
     keywords="lbry protocol media",
     license='MIT',
-    python_requires='>=3.6',
+    python_requires='>=%d.%d.%d' % (__min_python_version__[0], __min_python_version__[1], __min_python_version__[2]),
```

Comment:
`f">={'.'.join(__min_python_version__)}"` is cleaner.

Reply:
python is nuts :-)